src/transformState.js

```diff
@@ -62,7 +62,27 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  const transformed = state;
```

Comment:
Незрозуміло, що саме `transformed`
```suggestion
  const copiedState = state;
```